scripts/tests/cirque_tests.sh

```diff
@@ -34,6 +34,7 @@ OT_SIMULATION_CACHE="$CIRQUE_CACHE_PATH/ot-simulation.tgz"
 
 # Append test name here to add more tests for run_all_tests
 CIRQUE_TESTS=(
+    "EchoTest"
```

Comment:
Please try to add InteractionModel test as well.

Reply:
ack